node/tests/p2p_test.go

```diff
@@ -37,3 +49,59 @@ func TestUseBridgeNodeAsBootstraper(t *testing.T) {
 		assert.True(t, nodes[index].Host.Network().Connectedness(addr.ID) == network.Connected)
 	}
 }
+
+/*
+Test-Case: Sync a Light Node with a Bridge Node
+Steps:
+1. Create a Bridge Node(BN)
+2. Start a BN
+3. Create full/light nodes with bridge node as bootsrapped peer
+4. Start full/light nodes
+5. Ensure that nodes are connected to bridge
+6. Check that full and light nodes are connected to each other
+*/
+func TestBootstrapNodesFromBridgeNode(t *testing.T) {
+	sw := swamp.NewSwamp(t)
+	cfg := node.DefaultConfig(node.Bridge)
+	cfg.P2P.Bootstrapper = true
+	bridge := sw.NewBridgeNode(node.WithConfig(cfg), node.WithRefreshRoutingTablePeriod(time.Second*30))
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	t.Cleanup(cancel)
+
+	err := bridge.Start(ctx)
+	require.NoError(t, err)
+	addr := host.InfoFromHost(bridge.Host)
+
+	full := sw.NewFullNode(
+		node.WithBootstrappers([]peer.AddrInfo{*addr}),
+		node.WithRefreshRoutingTablePeriod(time.Second*30),
+	)
+	light := sw.NewLightNode(
+		node.WithBootstrappers([]peer.AddrInfo{*addr}),
+		node.WithRefreshRoutingTablePeriod(time.Second*30),
+	)
+	nodes := []*node.Node{full, light}
+	for index := range nodes {
+		require.NoError(t, nodes[index].Start(ctx))
+		assert.Equal(t, *addr, nodes[index].Bootstrappers[0])
+		assert.True(t, nodes[index].Host.Network().Connectedness(addr.ID) == network.Connected)
+	}
+	sub0, err := light.Host.EventBus().Subscribe(&event.EvtPeerConnectednessChanged{})
```

Comment:
why sub0 🧐 ? 

![sz_comp](https://user-images.githubusercontent.com/1822365/173040912-bb97048c-f0d5-4105-ab4e-634b70566704.jpg)

Reply:
There were his brothers - sub1 and sub2. But they were destroyed 😄